ui/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala

```diff
@@ -47,6 +48,15 @@ case class RemoteEnvironmentConfig(targetEnvironmentId: String)
 case class EnvironmentAlert(content: String, cssClass: String)
 case class CommentSettings(matchExpression: String, link: String)
 case class DeploySettings(requireComment: Boolean)
+case class IntervalSettings(base: Int, processes: Int, healthCheck: Int)
```

Comment:
Maybe it's better to put Options here? So that we can override only some settings?

And maybe name like ProcessListRefreshPeriodSettings? Interval can mean a lot of different things ;)

Reply:
I'v changed it to IntervalTimeSettings - now it's okey.

Reply:
kind of better, but still doesn't look like frontend refresh period...